content/en/service_management/incident_management/incident_settings/notification_rules.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Notification Rules
+further_reading:
+- link: "/service_management/incident_management/incident_settings/templates"
+  tag: "Documentation"
+  text: "Customize message templates"
+---
+
+## Overview
+
+Automated, customizable notification rules ensure the right stakeholders are alerted based on specific criteria. This removes the burden from the individual declaring the incident and ensures prompt involvement of the necessary parties, expediting the resolution process. For example, Set a notification rule to automatically notify team stakeholders whenever a SEV-1 or SEV-2 for `service:web-store` AND `application:purchasing` incident is declared and when that incident moves through different states of progression.
```

Comment:
```suggestion
Automated, customizable notification rules ensure the right stakeholders are alerted based on specific criteria. This removes the burden from the individual declaring the incident and ensures prompt involvement of the necessary parties, expediting the resolution process. For example, you can set a notification rule to automatically notify team stakeholders whenever a SEV-1 or SEV-2 incident for `service:web-store` AND `application:purchasing` is declared and when that incident moves through different states of progression.
```

i feel like this ordering of words is easier to parse, hopefully I did not change any meanings